plugins/autodim.koplugin/main.lua

```diff
@@ -204,6 +207,19 @@ function AutoDim:onSuspend()
     end
 end
 
+function AutoDim:updateFooter(clear)
+    -- update footer only if it is not covered by another widget
+    if self.top_widget_before_dim == "ReaderUI" or
+        (self.top_widget_before_dim ~= "ConfigDialog" and self.top_widget_before_dim ~= "ScreenSaver"
+        and self.top_widget_before_dim ~= "VirtualKeyboard") then
+
+        UIManager:broadcastEvent(Event:new("UpdateFooter", self.view and self.view.footer_visible or true))
```

Comment:
Possibly the fallback ought to be `false`, rather than `true`?

Reply:
Although this does no harm "false" is better.